builtin/game/chatcommands.lua

```diff
@@ -225,6 +225,19 @@ core.register_chatcommand("revoke", {
 			end
 		end
 		core.set_player_privs(revoke_name, privs)
+
+		--Check if we succeeded
```

Comment:
Can you put a space after the `--`? Its mandated by the lua style guidelines: http://dev.minetest.net/Lua_code_style_guidelines